heroes.json

```
@@ -329,6 +329,7 @@
   {"x": 2, "y": 8, "color": "#2bc9ad", "userid": "ryan-seedlink"},
   {"x": 6, "y": 15, "color": "#3780E9", "userid": "reallEz"},
   {"x": 2, "y": 11, "color": "#02a2ff", "userid": "guanzhaoheng"},
-  {"x": 14, "y": 9, "color": "#02a2ff", "userid": "GGGGGHT"}
+  {"x": 14, "y": 9, "color": "#02a2ff", "userid": "GGGGGHT"},
+  {"x": 4, "y": 7, "color": "#02a2ff", "userid": "GGGGGHT"}
```

请将`userid`替换为你自己的id。
```suggestion
  {"x": 4, "y": 7, "color": "#02a2ff", "userid": "Camille-seven"}
```